src/uct/base/uct_iface.c

```diff
@@ -553,37 +553,39 @@ ucs_config_field_t uct_iface_config_table[] = {
 ucs_status_t uct_base_ep_am_short_iov(uct_ep_h ep, uint8_t id, const uct_iov_t *iov,
                                       size_t iovcnt)
 {
+    uint64_t header = 0;
+    size_t offset   = 0;
```

Comment:
can have a single variable for `payload_length` and `offset`, e.g. `size_t length` and rename `length` for buffer to `buffer_length`

Reply:
ok